### PR TITLE
docs: Add 'Writing Your Own Policies' tutorial

### PR DESCRIPTION
**Pull request by shimonp21**

### CODE REVIEW

File: docs/tutorials/policies/getting-started-with-policies.mdx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+```bash
+cloudquery policy run aws//foundational_security
+```
+
+or just the EC2 section with:
+```bash
+cloudquery policy run aws//foundational_security/ec2
+```
+
+Running prewritten policies is all well and good, but how to write your own policies and queries?
```
Comment (yevgenypats) [inline docs/tutorials/policies/getting-started-with-policies.mdx:67]: I'd keep it more professional. Just Writing your own policy.

File: docs/tutorials/policies/policies-overview.mdx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+The CloudQuery team and community already implemented many different policies for many different providers.
+Among them are the [AWS CIS Benchmark](https://www.cisecurity.org/benchmark/amazon_web_services), 
+the [PCI DSS](https://www.pcisecuritystandards.org/), and 
+the [AWS Foundational Security Best Practices](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp.html).
+
+You can find the full list of existing policies on
+
+- [CloudQuery Hub](https://hub.cloudquery.io/) ([AWS](https://hub.cloudquery.io/policies/cloudquery/aws/latest/policies),
+[GCP](https://hub.cloudquery.io/policies/cloudquery/gcp/latest/policies), [Azure](https://hub.cloudquery.io/policies/cloudquery/azure/latest/policies)).
```
Comment (ukmadlz) [inline docs/tutorials/policies/policies-overview.mdx:37]:
```suggestion
[GCP](https://hub.cloudquery.io/policies/cloudquery/gcp/latest/policies), [Azure](https://hub.cloudquery.io/policies/cloudquery/azure/latest/policies))
```

File: docs/tutorials/policies/writing-your-first-policy.mdx [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+and run it with
+```bash
+cloudquery policy run ./my_first_policy
+```
+
+:::caution Important
+Due to the fact that the `cloudquery policy run` also supports running policies from [our official github](https://github.com/cloudquery-policies),
+to run policies from the local filesystem, you **must** specify a path beginning with `./` or `../` -
+absoute pats will not work**
```
Comment (roneli) [inline docs/tutorials/policies/writing-your-first-policy.mdx:39]:
```suggestion
absolute paths will not work**
```

File: docs/tutorials/policies/writing-your-first-policy.mdx [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+:::
+
+Let's go over the structure of this `policy.hcl` file:
+- At the root is the `policy` block, followed by its label.
+- The `title` is a human readable description of the policy.
+- The mandatory `configuration` block defines which CloudQuery providers are required to run this policy.
+  For instance, if you are querying aws tables, you must have an `aws` provider block in your `configuration`.
+  - The `version` field of the `provider` block is a string describing which version of the CloudQuery provider to use.
+  - You can have multiple `providers` in this section, if your policy queries several different clouds.
+- Next, the `check` block describes the "meat" of the policy - which checks to run.
```
Comment (roneli) [inline docs/tutorials/policies/writing-your-first-policy.mdx:49]: nit: different word for "meat" perhaps.

File: docs/tutorials/policies/writing-your-first-policy.mdx [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+absoute pats will not work**
+:::
+
+Let's go over the structure of this `policy.hcl` file:
+- At the root is the `policy` block, followed by its label.
+- The `title` is a human readable description of the policy.
+- The mandatory `configuration` block defines which CloudQuery providers are required to run this policy.
+  For instance, if you are querying aws tables, you must have an `aws` provider block in your `configuration`.
+  - The `version` field of the `provider` block is a string describing which version of the CloudQuery provider to use.
+  - You can have multiple `providers` in this section, if your policy queries several different clouds.
```
Comment (roneli) [inline docs/tutorials/policies/writing-your-first-policy.mdx:48]: its only mandatory in the root policy.

File: docs/tutorials/policies/views.mdx [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+For this tutorial, We'll write a policy that finds all AWS EC2 instances that have unrestricted 
+outbound access to the internet (or more specifically, all EC2 instances that have a security group that allows
+unrestricted access to the internet). This example is inspired by our 
+[public egress policy](https://github.com/cloudquery-policies/aws/tree/main/public_egress) that we released
+in response to the [log4j vulnerability](https://www.govcert.ch/blog/zero-day-exploit-targeting-popular-java-library-log4j).
+The rationale is that for an attacker to exploit the log4j vulnerabiliy, the exploited server must have outbound 
+access to the internet (so the server can download the malicious payload). 
+If your server doesn't have such access, an attacker is less likely to be able to exploit the log4j vulnerability.
+Of course, it is still of critical importance to patch all vulnerable java applications ASAP. 
+You can read more about the log4j vulnerability and the `public_egress` poicy int [this blog post](https://www.cloudquery.io/blog/aws-log4j-and-finding-unrestricted-outbound-access).
```
Comment (roneli) [inline docs/tutorials/policies/views.mdx:15]: you explain the rationale of log4j, but I was expecting the rationale of why we should use views, which is to reduce query complexity, specially if the are multiple checks that would benefit from it.

File: docs/tutorials/policies/subpolicies.mdx [hunk 7]
```diff
@@ -0,0 +1,158 @@
...
+So, to run the `foundational_security` **subpolicy** in the the `aws` **policy**, we run.
+
+```bash
+cloudquery policy run aws//foundational_security
+```
+
+```tip title="Summary"
+When calling `cloudquery policy run`, the `//` separator separates the "path to the root policy" from
+the "path to the subpolicy/subquery". It should always appear **after the root poicy**.
+```
```
Comment (roneli) [inline docs/tutorials/policies/subpolicies.mdx:158]: this didn't render nicely.

**Review by roneli — CHANGES_REQUESTED** [review]
Added some comments.

File: docs/tutorials/policies/writing-your-first-policy.mdx (hunk 4 above)
Comment (shimonp21) [reply docs/tutorials/policies/writing-your-first-policy.mdx:49]: How About "core"?

Comment (shimonp21) [reply docs/tutorials/policies/writing-your-first-policy.mdx:49]: how about "core"?

File: docs/tutorials/policies/writing-your-first-policy.mdx (hunk 5 above)
Comment (shimonp21) [reply docs/tutorials/policies/writing-your-first-policy.mdx:48]: Removed the word "mandatory"

File: docs/tutorials/policies/views.mdx (hunk 6 above)
Comment (shimonp21) [reply docs/tutorials/policies/views.mdx:15]: Agreed - removed most of the explanations, leaving only the link to the blog.
Added another sentence about the benefits of using views.

File: docs/tutorials/policies/subpolicies.mdx (hunk 7 above)
Comment (shimonp21) [reply docs/tutorials/policies/subpolicies.mdx:158]: Fixed

**Pull request merged**